src/poliastro/plotting.py

```diff
@@ -380,18 +380,26 @@ def savefig(self, filename, **layout_kwargs):
         )
 
 
-def plot_solar_system(epoch=None):
+def plot_solar_system(outer=True, epoch=None):
     """
     Plots the whole solar system in one single call.
 
     Parameters
     ------------
 
-    Epoch Value. By default is None.
+    outer : To print only the inner solar system , pass outer=False else it will print the whole solar system
+    epoch: Epoch Value of the plot. By default the value is None.
     """
-    orbits = [Earth, Mars, Mercury, Jupiter, Venus, Saturn, Uranus, Neptune]
-    op = OrbitPlotter()
-    for orbit in orbits:
-        orb = Orbit.from_body_ephem(orbit)
-        op.plot(orb)
+    if outer:
+        orbits = [Earth, Mars, Mercury, Jupiter, Venus, Saturn, Uranus, Neptune]
```

Comment:
I think the `OrbitPlotter` and the `for` loop in `orbits` can be made outside the `if` block, to avoid repeating it.